### PR TITLE
Added pagination to project slippi

### PR DESCRIPTION
**Pull request by hunkim2**
-Added pagination

### CODE REVIEW

File: Launcher/app/components/stats/PunishesTable.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import classNames from 'classnames';
 import React, { Component } from 'react';
 import { Table, Image, Icon } from 'semantic-ui-react';
 
-import styles from './GameProfile.scss';
+import styles from '../../styles/pages/GameProfile.scss';
```
Comment (JLaferri) [inline Launcher/app/components/stats/PunishesTable.js:6]: I think I would still prefer the css be co-located with the component.

File: Launcher/app/reducers/fileLoader.js [hunk 2]
```diff
@@ -114,11 +126,29 @@ function changeFolderSelection(state, action) {
       game: game,
     };
   });
+  const numberOfPages = Math.ceil(files.length / 10);
+  const paginatedFiles = [];
+  for (let pagesX = 0; pagesX <= numberOfPages; pagesX++) {
```
Comment (JLaferri) [inline Launcher/app/reducers/fileLoader.js:131]: This loop could just be `_.chunk` I think?

File: Launcher/app/containers/FileLoader.js [hunk 3]
```diff
@@ -29,34 +43,49 @@ export default class FileLoader extends Component {
   };
 
   refPrimary: {};
-
+  constructor(props) {
+    super(props);
+    this.state = {
```
Comment (JLaferri) [inline Launcher/app/containers/FileLoader.js:48]: Could current page be stored in redux?

File: Launcher/app/containers/FileLoader.js [hunk 4]
```diff
@@ -222,3 +274,23 @@ export default class FileLoader extends Component {
     );
   }
 }
+
+function mapStateToProps(state) {
```
Comment (JLaferri) [inline Launcher/app/containers/FileLoader.js:278]: I feel like I might still prefer this stuff being in a different file but co-located with the component. Also I feel like they should both be stored within their own folder and expose an `index.js`. I can take a crack at organizing stuff this way if you like.

**Review by JLaferri — CHANGES_REQUESTED** [review]
Overall this looks quite good. There's just a few things I'm thinking could be changed.

1) File structure
I think I might take a stab at setting this up differently if you're cool with that. I mentioned some of my thoughts on it in the code comments.

2) Pagination Styling
This is the current styling correct?
image

I think I would prefer it to have the look like the other example:
image

Also not a huge fan of the glowing effect.

File: Launcher/app/reducers/fileLoader.js (hunk 2 above)
Comment (hunkim2) [reply Launcher/app/reducers/fileLoader.js:131]: I've never used chunk before. Is that lodash?

File: Launcher/app/containers/FileLoader.js (hunk 3 above)
Comment (hunkim2) [reply Launcher/app/containers/FileLoader.js:48]: Yes it can be, I can change that then.

File: Launcher/app/containers/FileLoader.js (hunk 4 above)
Comment (hunkim2) [reply Launcher/app/containers/FileLoader.js:278]: I agree with you that component based files should be directed into their own folder system with an exposed `index.js` that handles the routing and store preferences.

**Pull request closed without merging**